assets/js/components/notifications/FirstPartyModeSetupBanner.js

```diff
@@ -60,10 +67,20 @@ export default function FirstPartyModeSetupBanner( { id, Notification } ) {
 		select( CORE_NOTIFICATIONS ).isNotificationDismissed( id )
 	);
 
+	const { dismissNotification, invalidateResolution } =
+		useDispatch( CORE_NOTIFICATIONS );
+	const { setValue } = useDispatch( CORE_UI );
+
 	const onCTAClick = () => {
 		setFirstPartyModeEnabled( true );
 		saveFirstPartyModeSettings();
```

Comment:
Should we consider (a)waiting for this fetch to complete, and only proceed if there is no error?

Reply:
Thanks for raising that - I was meaning to create an issue to add error handling for this component. I've done so now: https://github.com/google/site-kit-wp/issues/9846